lib/spotlight/dor/resources/version.rb

```diff
@@ -2,7 +2,7 @@ module Spotlight
   module Dor
     # :nodoc:
     module Resources
-      VERSION = '0.4.1'
+      VERSION = '0.5.0'
```

Comment:
Yeah -- once we release this, I have a commit to sul-dlss/exhibits to update it to spotlight 0.16.